sdk/analytics/src/main/java/com/klaviyo/analytics/networking/KlaviyoApiClient.kt

```diff
@@ -152,13 +152,13 @@ internal object KlaviyoApiClient : ApiClient {
             } catch (exception: JSONException) {
                 wasMutated = true
                 Registry.log.wtf("Invalid persistent queue JSON", exception)
-                Registry.log.debug(it)
+                Registry.log.info(it)
```

Comment:
Just noting this jumped from debug to info whereas other jumps were maybe not as severe _I think_.

Reply:
You're correct. Almost all cases I reduced log level. But there were two cases where we had a `wtf` error followed by some additional explanatory data. I thought in those cases `info` made more sense.